### PR TITLE
Default rinkeby to not connect to sequencer

### PR DESCRIPTION
**Pull request by joshuacolvin0**
The following settings have new defaults for rinkeby testnet because classic does not connect to nitro sequencer
* `--feed.input.url= --node.forwarder.rpc-mode=non-mutating`

Setting `feed.input.url` to nothing disables connecting to sequencer feed. Setting `node.forwarder.rpc-mode` to `non-mutating` disables connecting to the sequencer RPC.

### CODE REVIEW

File: packages/arb-rpc-node/aggregator/aggregator.go [hunk 1]
```diff
@@ -161,22 +165,35 @@ func (m *Server) LatestSnapshot(ctx context.Context) (*snapshot.Snapshot, error)
...
+			return nil, err
+		}
+		if pending == nil {
+			return m.LatestSnapshot(ctx)
+		}
+		return pending, nil
 	}
-	return pending, nil
+
+	return nil, errors.New("no batcher defined, cannot generate pending snapshot")
```
Comment (PlasmaPower) [inline packages/arb-rpc-node/aggregator/aggregator.go:179]: We should return `m.LatestSnapshot(ctx)` in this case

**Review by PlasmaPower — COMMENTED** [review]
Small comment but generally LGTM

File: packages/arb-rpc-node/aggregator/aggregator.go (hunk 1 above)
Comment (joshuacolvin0) [reply packages/arb-rpc-node/aggregator/aggregator.go]: :+1:

**Review by PlasmaPower — APPROVED** [review]
LGTM

**Pull request merged**